pkg/kubelet/prober/prober.go

```diff
@@ -165,31 +162,31 @@ func (pb *prober) runProbe(probeType probeType, p *v1.Probe, pod *v1.Pod, status
 		return pb.exec.Probe(pb.newExecInContainer(container, containerID, command, timeout))
```

Comment:
Is `buildHeader` above (line 152) still used?

Reply:
No. I'll remove it.